slf4j-tinylog/src/main/java/org/tinylog/slf4j/TinylogSlf4jServiceProvider.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 Martin Winandy
```

Comment:
Please update the year to 2021. You can also replace my name by your's if you want.

Reply:
Will do. That's not really important to me, but thanks for suggesting, that's kind 😃